easybuild/easyconfigs/p/pkg-config/pkg-config-0.29-foss-2015b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'pkg-config'
+version = '0.29'
+
+homepage = 'http://www.freedesktop.org/wiki/Software/pkg-config/'
+description = """pkg-config is a helper tool used when compiling applications and libraries. It helps you insert the
+ correct compiler options on the command line so an application can use
+  gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
+ for instance, rather than hard-coding values on where to find glib (or other libraries)."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://pkgconfig.freedesktop.org/releases/']
```

Comment:
@nathanhaigh Existing `pkg-config` easyconfigs use `https://pkg-config.freedesktop.org/releases/`, so please do so here as well?

Also, please include SHA256 checksums in all easyconfigs. EasyBuild can do that for you via `eb *.eb --inject-checksums`.